paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -829,7 +831,7 @@ class PaparazziPluginTest {
     assertThat(resourceFileContents[1]).isEqualTo("intermediates/merged_res/debug")
     assertThat(resourceFileContents[4]).isEqualTo("intermediates/assets/debug")
     assertThat(resourceFileContents[5]).isEqualTo("app.cash.paparazzi.plugin.test,com.example.mylibrary,app.cash.paparazzi.plugin.test.module1,app.cash.paparazzi.plugin.test.module2")
-    assertThat(resourceFileContents[6]).isEqualTo("src/main/res,src/debug/res")
+    assertThat(resourceFileContents[6]).isEqualTo("build/generated/res/resValues/debug,src/debug/res,src/main/res")
```

Comment:
mind the order of `debug` vs `main`: https://github.com/cashapp/paparazzi/pull/980/files#r1254387829

It's important because of [variant folder precedence](https://developer.android.com/build/build-variants#sourceset-build), I think.

Related (depending on where the reversal happens): https://github.com/cashapp/paparazzi/pull/980#discussion_r1254159048

Reply:
@kevinzheng-ap, can you please explain in why the order matters (if it does)?

(From the earlier PR I understood it's because Paparazzi/layoutlib is emulating a full-on Android build/runtime, so the folder order determines which resources are loaded at runtime. Consider having a same named resource in `src/main/res/values/x.xml` and `src/debug/res/values/x.xml` and what'll be rendered on the screenshot vs on device.)

> not something we can change anyway

It's up to Paparazzi how it interprets and uses the AGP APIs.